packages/editor-ui/vite.config.mts

```diff
@@ -73,8 +74,11 @@ const plugins = [
 	icons({
 		compiler: 'vue3',
 	}),
-	vue()
+	vue(),
 ];
+if (process.env.ENABLE_TYPE_CHECKING === 'true') {
```

Comment:
could we add a command on package.json to run with this enabled?

Reply:
considering that (hopefully) this is temporary, anyone wanting to use this can use `ENABLE_TYPE_CHECKING=true pnpm build`. I don't see the value in adding an alias for this.

Reply:
I was thinking it's a good way to document this and point people to it. Could we add warnings to the normal build? or dev build? I think this would really push us to fix these quickly

Reply:
I could not figure a way out to show these as warning. When the flag is enabled, the build simply fails.

Reply:
alright thank you for looking into this. I am just hoping for a way to make this visible to everyone.. alternatively maybe a non-required Github action that builds with this 🤔..

Reply:
Or github build that runs on cron and reports weekly to Slack webhook..